docs/examples/driver_examples/Qcodes example with Lakeshore 325.ipynb

```diff
@@ -3645,6 +3645,76 @@
     "lake.heater_1.setpoint(15.0)  # <- temperature \n",
     "live_plot_temperature_reading(lake.sensor_a, n_reads=400)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Querying the resistance and heater output "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "25"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# to get the resistance of the system (25 or 50 Ohm)\n",
+    "lake.heater_1.resistance()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "50"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# to set the resistance of the system (25 or 50 Ohm)\n",
+    "lake.heater_1.resistance(50)\n",
+    "lake.heater_1.resistance()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.0"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lake.heater_1.heater_output() # in %, 50 means 50%"
```

Comment:
please add a clarification as in the instrument's manual: smth around the lines of "heater output in percent (%) of current or power, depending on setting."

Reply:
Clarification has been added, together with instruction how to get the setting value.